app/src/main/java/com/islaharper/dawnofandroid/presentation/screens/login/LoginContent.kt

```diff
@@ -0,0 +1,105 @@
+package com.islaharper.dawnofandroid.presentation.screens.login
+
+import android.content.res.Configuration
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.LinearProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.islaharper.dawnofandroid.R
+import com.islaharper.dawnofandroid.domain.model.MessageBarState
+import com.islaharper.dawnofandroid.presentation.common.GoogleButton
+import com.islaharper.dawnofandroid.presentation.components.MessageBar
+
+@Composable
+fun LoginContent(
+    modifier: Modifier = Modifier,
```

Comment:
Can we please check the parameter order for these Composables? There is a set of lint rules you can bring into the project to help check for these if you are interested, https://slackhq.github.io/compose-lints/

Let me know if you need a hand, I tried it in another project and it can be a little tricky to get set up :+1:

Reply:
I added ktlint and detekt to the project, perhaps it needs further setting up?

Reply:
I think it needs to be added as a `lintCheck(` in the build.gradle

https://slackhq.github.io/compose-lints/